README.md

```diff
@@ -89,30 +112,32 @@ kubectl create namespace west
 kubectl config set-context --current --namespace west
 ~~~
 
-## Step 3: Set up your Podman environment
+## Step 4: Set up your Podman environment
 
 Open a new terminal window and set the `SKUPPER_PLATFORM`
 environment variable to `podman`.  This sets the Skupper platform
 to Podman for this terminal session.
 
-Use `systemctl` to enable the Podman API service.
+The `skupper system install` enables the Podman API
+service (using systemctl) if that service is not already running.
+The `skupper system install` command also creates the
+Skupper controller container for the current user.
 
 _**Podman:**_
 
 ~~~ shell
 export SKUPPER_PLATFORM=podman
```

Comment:
I feel the intro to the different shells isn't as clear in this example. Not a blocker, but i'd like to look into it further